### PR TITLE
Inverted page curl

### PR DESCRIPTION
**Pull request by gunrot**

### CODE REVIEW

File: transitions/InvertedPageCurl.glsl [hunk 1]
```diff
@@ -0,0 +1,214 @@
+// author: Hewlett-Packard
+// license: bsd 3-clause
```
Comment (gltransitions) [inline transitions/InvertedPageCurl.glsl]: 'bsd 3-clause' not found in supported licenses: MIT, EPL-1.0, CDDL-1.0, MPL-2.0, LGPL, GPLv2, GPLv3, Apache-2.0, BSD 2 Clause

Comment (gre) [reply transitions/InvertedPageCurl.glsl]: can it be "BSD 2 Clause" or do I need to add BSD 3? ^^

Comment (gre) [reply transitions/InvertedPageCurl.glsl]: i'm adding it anyway

Comment (gunrot) [reply transitions/InvertedPageCurl.glsl]: Well 3 clause is not the same as 2 clause ;-) I did not choose the license.

Comment (gre) [reply transitions/InvertedPageCurl.glsl]: ok I should have fixed it , can you rename it to exactly `BSD 3 Clause` ?

Comment (gunrot) [reply transitions/InvertedPageCurl.glsl]: I try

**Pull request merged**